aws/private/py_lambda.bzl

```diff
@@ -18,99 +15,78 @@ def _short_path(file_):
         short_path = short_path[second_slash + 1:]
     return short_path
 
+# Copied from aspect-bazel-lib/lib/private/tar.bzl
+def _mtree_line(file, type, content = None, uid = "0", gid = "0", time = "1672560000", mode = "0755"):
```

Comment:
we should match the defaults of pkg_tar, mode would be `0644` in this case, if we want them to be identical.

Reply:
Yeah that explains why I was able to simplify and not treat bin specially.